web/src/app/schedules/temp-sched/TempSchedShiftsList.tsx

```diff
@@ -147,23 +143,26 @@ export default function TempSchedShiftsList({
             ),
             userID: s.userID,
             icon: <UserAvatar userID={s.userID} />,
+            disabled: isHistoricShift,
             secondaryAction:
               index === 0 ? (
                 <div className={classes.secondaryActionWrapper}>
-                  {!isValid && (
+                  {!isValid && !isHistoricShift && (
                     <Tooltip
                       title='This shift extends beyond the start and/or end of this temporary schedule'
                       placement='left'
                     >
                       <Error color='error' />
                     </Tooltip>
                   )}
-                  <IconButton
-                    aria-label='delete shift'
-                    onClick={() => onRemove(s)}
-                  >
-                    <Delete />
-                  </IconButton>
+                  {isHistoricShift ? null : (
```

Comment:
This could also be written as
```
{!isHistoricShift && (
  <IconButton
    aria-label='delete shift'
    onClick={() => onRemove(s)}
  >
    <Delete />
  </IconButton>
)}
```